atomic_cms.gemspec

```diff
@@ -4,10 +4,10 @@ Gem::Specification.new do |s|
   s.summary     = 'Atomic CMS'
   s.description = 'Live CMS powered by atomic assets.'
   s.authors     = ['Don Humphreys', 'Spartan']
```

Comment:
Probably want to update this one while we're here. Unless it'll mess with our RubyGems account?